.github/ISSUE_TEMPLATE/documentation_improvement.yaml

```diff
@@ -0,0 +1,40 @@
+name: Documentation Improvement
+description: Report wrong or missing documentation
+title: "DOC: "
+labels: [Docs, Needs Triage]
+
+body:
+  - type: checkboxes
+    attributes:
+      options:
+        - label: >
+            I have checked that the issue still exists on the latest versions of the docs
+            on `master` [here](https://pandas.pydata.org/docs/dev/)
+          required: true
+  - type: textarea
+    id: location
+    attributes:
+      label: Location of the documentation
+      description: >
+        This should provide the location of the documentation, e.g. "pandas.read_csv" or the
```

Comment:
Nit: Maybe for these descriptions `This should` -> `Please`

Reply:
Good call, have updated